src/rt-app_parse_config.c

```diff
@@ -971,6 +1046,9 @@ parse_task_data(char *name, struct json_object *obj, int index,
 			data->loop = -1;
 	}
 
+	/* Reset thread's current sched_data/taskgroup after parsing. */
+	data->curr_sched_data = NULL;
```

Comment:
ok you reset curr_sched_data at the end of the parsing

Reply:
That's right.